src/crystal/system/unix/time.cr

```diff
@@ -54,9 +54,11 @@ module Crystal::System::Time
   end
 
   def self.load_localtime : ::Time::Location?
-    if ::File.exists?(LOCALTIME)
+    if ::File.exists?(LOCALTIME) && ::File.readable?(LOCALTIME) && ::File.file?(LOCALTIME)
```

Comment:
`::File.exists?(LOCALTIME)` part is redundant, as `::File.file?(LOCALTIME)` returns `false` for nonexistent files.
 
```suggestion
    if ::File.file?(LOCALTIME) && ::File.readable?(LOCALTIME)
```